.github/workflows/pr_agent.yml

```
@@ -0,0 +1,24 @@
+name: AI PR Review
+
+on:
+  pull_request:
+
+jobs:
+  review:
+    runs-on: ubuntu-latest 
+    steps:
+      - uses: actions/checkout@v3
```

[ReviewCategory.STYLE] The `actions/checkout@v3` action is an older version. The latest stable version is `v4`. It is recommended to use the latest version to receive bug fixes and new features.

```suggestion
      - uses: actions/checkout@v4
```